packages/serverpod/lib/src/server/future_call.dart

```diff
@@ -6,12 +6,15 @@ import 'session.dart';
 /// Superclass of a [FutureCall], override the [invoke] method to create a
 /// custom [FutureCall]. The call also needs to be registered with the top
 /// [ServerPod] object before starting the [Server].
-abstract class FutureCall {
+abstract class FutureCall<DataType extends SerializableEntity?> {
```

Comment:
Thought it might be nice to allow `FutureCalls` to take no args (`class TestCall2 extends FutureCall<Null>`) or a nullable arg (`class TestCall extends FutureCall<SimpleData?>`) or a nonnull arg (`class TestCall extends FutureCall<SimpleData>`). So it is intentional, but we can remove it as well.

Reply:
DataType -> T: 03364b7

Reply:
I think it's generally better to keep the nullability out of the generic types where possible. Makes the code more readable as it is not otherwise immediately clear that e.g. `Future<void> invoke(Session session, T object);` supports null values.